### DISCUSSION TITLE
Slack auth problem

### DISCUSSION THREAD

**Discussion by mweth** [outline/outline#4477] [Self Hosting] [answered]
### Discussed in https://github.com/outline/outline/discussions/3671

<div type='discussions-op-text'>

<sup>Originally posted by **Comandante1977** June 18, 2022</sup>
Hi, everybody!

Another problem with my self-hosting installation. 
Everything seems to begin working, but I've got the Slack auth error.
The first page ok:
![image](https://user-images.githubusercontent.com/45628795/174445174-6bf4d72b-640a-421e-b838-c96695d493d6.png)
And then:
![image](https://user-images.githubusercontent.com/45628795/174445199-35ffbb06-be7a-4df1-9310-9be29d4e355e.png)

All credentials are in place in .env file.
I'm not Slack admin, so may be admin did smth wrong.

In .env I have these lines configured:

https://outline.yourdomain.com/auth/slack.callback
SLACK_KEY=Slack Client ID value
SLACK_SECRET=Slakc Client Secret
SLACK_VERIFICATION_TOKEN=Slack Verification Token
SLACK_APP_ID=Slack App ID

Any ideas?
</div>

**Comment by tommoor** [comment, marked as answer]
The answer is right there in your screenshot – 
![image](https://user-images.githubusercontent.com/380914/203813985-bc2b73e9-f356-4cf9-b23d-df1e54bc0b3b.png)
